Persephone/AppDelegate.swift

```diff
@@ -34,22 +31,16 @@ class AppDelegate: NSObject,
         $0.uiState
       }
     }
+    
+    instantiateControllers()
+    connectToMPDServer()
   }
 
   func connectToMPDServer() {
-    let mpdServer = App.store.state.preferencesState.mpdServer
-
-    App.mpdClient = MPDClient(
-      host: mpdServer.hostOrDefault,
-      port: mpdServer.portOrDefault,
-      withDelegate: App.mpdServerDelegate
-    )
-
-    App.mpdClient.connect()
+    App.mpdServerController.connect()
```

Comment:
Since this function is now only housing one other function, what would you think about having `applicationDidFinishLaunching()` call `App.mpdServerController.connect()` directly?

Reply:
And if not -- should it be `private`?